code/alexander/python/oop/rectangle_class.py

```diff
@@ -0,0 +1,15 @@
+class rectangle:
+    def __init__(self, length, width):
+        self.length = length
+        self.width = width
+    def perimeter(self):
+        return (self.length * 2) + (self.width * 2)
+    def area(self):
+        return self.length * self.width
+    def display(self):
+        print(self.length, self.width, self.perimeter, self.area)
+class parallelepipede(rectangle):
+    def __init_(self, height):
```

Comment:
When you add the height value to the parallelepiped class you're not calling the super method correctly. Needs correction:

class Rabbit(Animal):
    def __init__(self, legs, color, speed): ##adding the new attribute speed
        super().__init__(legs, color) ## referencing old attributes legs,color
        self.speed = speed

Reply:
Hey can you verify I corrected this properly. should be good to go. just want to make sure.